gtda/mapper/visualization.py

```diff
@@ -19,7 +19,9 @@ def plot_static_mapper_graph(
         color_variable=None, node_color_statistic=None,
         color_by_columns_dropdown=False, plotly_kwargs=None,
         clone_pipeline=True):
-    """Plotting function for static Mapper graphs.
+    """Plotting function for static Mapper graphs, with nodes colored according
```

Comment:
I think we still want the one-line description as sphinx knows to only display the first paragraph in the index page. This way, it will be too long. So I suggest a one-liner plus another paragraph with more detail.

Reply:
Good catch! I've also added similar information to the interactive docstring.